force-app/main/default/classes/TestClassDeletion.cls

```diff
@@ -0,0 +1,34 @@
+@isTest(seealldata=false)
+public class TestClassDeletion {
+    @isTest static void testPositive(){
```

Comment:
Instead of `testPositive`, it should be something like `testUpdateClass_WithFemaleStudent`